packages/sources/finnhub/src/transport/quote-http.ts

```diff
@@ -3,6 +3,14 @@ import { BaseEndpointTypes, buildSymbol } from '../endpoint/quote'
 import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
 import { makeLogger } from '@chainlink/external-adapter-framework/util'
 
+export function parseResult(ticker: string, result: number) {
+  if (ticker == 'OANDA:WHEAT_USD') {
+    // if Ticket ONADA:WHEAT_USD convert result from dollar to cents
+    return result * 100
+  }
+  return result
+}
+
```

Comment:
Minor: Can we extract this out to a `transport/utils.ts` file seeing as it is common to both REST and WS?